helm-charts/daskhub/values.yaml

```diff
@@ -177,38 +196,118 @@ dask-gateway:
                     chars.append(escaped_hex_char)
             return u''.join(chars)
 
+        # Decide on available instance types and their resource allocation
+        # choices to expose based on cloud provider. For each daskhub hub
+        # managed by 2i2c, there should be these instance types available.
```

Comment:
Maybe add a link to the docs https://infrastructure.2i2c.org/topic/infrastructure/cluster-design/#for-nodes-where-user-servers-will-be-scheduled-on and a note that if these ever change this piece of code needs to also be changed?

Reply:
This is related to the dask worker nodes, which are undocumented to be systematically set at 16 CPU nodes.

Since this is a controversial topic, I'd like to not touch it further. Yuvi requested that #3307 use smaller nodes etc.

Reply:
Fair enough! Thank you!

Reply:
I'll open a dedicated PR to follow this up